exposed-java-time/src/main/kotlin/org/jetbrains/exposed/sql/javatime/JavaDateColumnType.kt

```diff
@@ -31,72 +31,77 @@ private val MYSQL_FRACTION_DATE_TIME_STRING_FORMATTER by lazy {
         Locale.ROOT
     ).withZone(ZoneId.systemDefault())
 }
-
-internal val ORACLE_TIME_STRING_FORMATTER by lazy {
+private val MYSQL_DATE_TIME_STRING_FORMATTER by lazy {
     DateTimeFormatter.ofPattern(
-        "1900-01-01 HH:mm:ss",
+        "yyyy-MM-dd HH:mm:ss",
         Locale.ROOT
-    ).withZone(ZoneOffset.UTC)
+    ).withZone(ZoneId.systemDefault())
 }
 
-private val MYSQL_TIME_AS_DEFAULT_STRING_FORMATTER by lazy {
+private val ORACLE_TIME_STRING_FORMATTER by lazy {
     DateTimeFormatter.ofPattern(
-        "HH:mm:ss",
+        "1900-01-01 HH:mm:ss",
```

Comment:
Could you tell me why we're using the 1900-01-01 string in the formatting pattern? I would write small KDoc about this